src/System.Private.CoreLib/shared/System/Reflection/Emit/Opcode.cs

```diff
@@ -183,9 +183,11 @@ public override int GetHashCode()
             return Value;
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
This can be removed once ValueType.ToString is fixed to return string?, right?

Reply:
Fixed.